docs/running.rst

```diff
@@ -1000,19 +1000,32 @@ If you now try to run a test that loads the module `cudatoolkit`, the following
      * Reason: caught framework exception: module cyclic dependency: cudatoolkit->foo->bar->foobar->cudatoolkit
    ------------------------------------------------------------------------------
 
-Flexible task allocation
-------------------------
+Controlling the Flexible Task Allocation
+----------------------------------------
 
 .. versionadded:: 2.15
 
-ReFrame can automatically determine the number of tasks used for a particular test.
-In order to instruct ReFrame that the number of tasks is going to be determined at runtime, :attr:`num_tasks <reframe.core.pipeline.RegressionTest.num_tasks>` has to be set to ``0``.
-This feature is used in conjuction with the ``--flex-alloc-tasks`` command line option.
-Therefore, if ``--flex-alloc-tasks`` is set to ``idle``, ReFrame is going to determine the number of tasks based on the idle nodes matching the ``access`` options of the logical partition (see `"Partition Configuration" <configure.html#partition-configuration>`__) as defined in the ``site_configuration`` dictionary.
-For ``--flex-alloc-tasks`` set to ``all``, ReFrame will determine the number of tasks based on the idle nodes respecting the command line options for ``partition``, ``reservation``, ``nodelist``, ``exclude-nodes`` and their ``job-option`` equivalents (see `"Controlling the Execution of Regression Tests" <running.html#controlling-the-execution-of-regression-tests>`__).
-For both ``idle`` and ``all`` the final number of tasks will be the product of the nodes satisfying the required criteria and :attr:`num_tasks_per_node <reframe.core.pipeline.RegressionTest.num_tasks_per_node>`.
-Finally, by specifying a positive integer ReFrame is going to use that number as the number of tasks without doing any additional check on the nodes.
+ReFrame can automatically set the number of tasks of a particular test, if its :attr:`num_tasks <reframe.core.pipeline.RegressionTest.num_tasks>` attribute is set to ``0``.
+By default, ReFrame will spawn such a test on all the idle nodes of the current system partition.
+This behavior can be adjusted using the ``--flex-alloc-tasks`` command line option.
+This option accepts three values:
+
+  1. ``idle``: (default) In this case, ReFrame will set the number of tasks to the number of idle nodes of the current logical partition.
```

Comment:
The number of tasks equals the number of nodes multiplied by the number of tasks per node. Therefore, it should  be mentioned here.

Reply:
Can you fix that?